grid_apps/AppManager.js

```diff
@@ -162,6 +162,7 @@ class AppManager extends EventEmitter {
 
     const { dependencies } = app
     if (dependencies) {
+      var sequence = Promise.resolve()
```

Comment:
this is an unusual pattern, so may be worth adding the stackoverflow link as a comment above this for context

Reply:
added, thanks